===================================================================
RCS file: /cvs/glibc/libc/io/sys/stat.h,v
retrieving revision 1.56
retrieving revision 1.57
diff -u -r1.56 -r1.57
--- libc/io/sys/stat.h	2007/05/10 23:01:02	1.56
+++ libc/io/sys/stat.h	2007/09/15 02:27:07	1.57
@@ -444,7 +444,7 @@
 		       __mode_t __mode, __dev_t *__dev)
      __THROW __nonnull ((3, 5));
 
-#if defined __GNUC__ && __GNUC__ >= 2 && defined __USE_EXTERN_INLINES
+#if defined __GNUC__ && __GNUC__ >= 2
 /* Inlined versions of the real stat and mknod functions.  */
 
 __extern_inline int
